test/unit/DCAStrategies/dca-strategies-positions-handler.spec.ts

```diff
@@ -82,6 +137,30 @@ contract('DCAStrategiesPositionsHandler', () => {
       then('transferFrom() is called correctly', async () => {
         expect(tokenA.transferFrom).to.have.been.calledOnceWith(user.address, DCAStrategiesPositionsHandlerMock.address, toDeposit);
       });
+      then('_approveHub() is called correctly', async () => {
+        expect(tokenA.approve.atCall(0)).to.have.been.calledWith(hub.address, constants.MAX_UINT_256);
```

Comment:
I know it's a pain, but let's do like we did with `_create`. Let's add an override on the mock, and check that `_approveHub` was called correctly, and only once